src/modules/Dashboard/hooks/useDecryptedIdentifiers.ts

```diff
@@ -20,6 +20,14 @@ export const useDecryptedIdentifiers = () => {
 
   return (identifiers: IdentifierResponse[]): Identifier[] =>
     identifiers.map(({ identifier, userPublicKey }) => {
+      // workaround for decrypted identifier data
```

Comment:
does it depend on a pending task on BE? When it should works properly?

Reply:
identifiers were not encrypted in legacy 
so we need to do an additional check on the client